x-pack/packages/security-solution/features/src/security/kibana_sub_features.ts

```diff
@@ -125,7 +125,7 @@ const trustedApplicationsSubFeature: SubFeatureConfig = {
     },
   ],
 };
-const hostIsolationExceptionsSubFeature: SubFeatureConfig = {
+const hostIsolationExceptionsSubFeatureBasic: SubFeatureConfig = {
```

Comment:
```suggestion
const hostIsolationExceptionsBasicSubFeature: SubFeatureConfig = {
```

Reply:
Changed in 701474a6f447d0f5270de971873d5133edece4fb